frankly034/Dantown
# #167961644 setup express

#### What does this PR do?
Setup basic express app with route versioning 

#### Description of Task to be completed?
- install express and dependencies
- setup express app
- setup route versioning

#### How should this be manually tested?
* clone the repo
* run `npm install`
* start the app with `npm run start:dev`
* visit `localhost:3000` to see the welcome message

#### Any background context you want to provide?
* must have `node` installed

#### What are the relevant pivotal tracker stories?
[Pivotal tracker story #167961644](https://www.pivotaltracker.com/n/projects/2385884/stories/167961644)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
